do_files_push.go

```diff
@@ -241,32 +245,58 @@ func doFilesPush(
 		response, err := client.UploadFile(project, request)
 
 		if err != nil {
-			return NewError(
-				hierr.Errorf(
-					err,
-					`unable to upload file "%s"`,
-					file,
-				),
+			if returnError(err) {
+				return NewError(
+					hierr.Errorf(
+						err,
+						`unable to upload file "%s"`,
+						file,
+					),
+
+					`Check, that you have enough permissions to upload file to`+
+						` the specified project`,
+				)
+			}
+			_, _ = os.Stderr.WriteString("Unable to upload file " + file + "\n")
```

Comment:
`\n` - how it will work for Win? It's strange that there is no existing method that writes string with newline. Pls use platform specific newline

Reply:
Done